app/views/task_mailer/task_deleted.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>A task in your project has been deleted!</h1>
+<p>The task "<%= @task.name %>" in the project "<%= @project.name %>" has been deleted.</p>
```

Comment:
You sender isn't worked because you don't set instance variable @task and @project in TaskMailer.task_deleted

```suggestion
<p>The task "<%= @task.name %>" in the project "<%= @project.name %>" has been deleted.</p>
```